### ISSUE TITLE
Upgrade to numpy 1.19.0 breaks time conversion functionality

### ISSUE THREAD

**Issue by jgliss** [metno/pyaerocom#176]
Not figured out what exactly goes wrong, but colocation tests fail online after recent numpy update to v1.19.0. This is likely related to conversion of NetCDF timestamps to numpy datetime64, and can probably be debugged via GriddedData.stop. 

All tests work locally using numpy 1.17.5.

**Please use numpy 1.17.5 until this has been clarified**

**Comment by jgliss** [comment]
@ejgal do you have time to look into this?

**Comment by ejgal** [comment]
@jgliss, I'll have a look!

**Comment by jgliss** [comment]
Fixed numpy=1.17.5 in pyaerocom_env.yml for now (branch v090dev).

**Comment by jgliss** [comment]
In `GriddedData.stop` `np.timedelta64` was invoked with input arg `unit` which is deprecated. And then there was a silly try/except block which I will remove for now and we see if this causes any issues in the future.

**Comment by jgliss** [comment]
I'll send a PR soon

**Comment by ejgal** [comment]
Yep, just found the same :)

**Comment by jgliss** [comment]
> Yep, just found the same :)

Sorry for making you look in parallel. I thought the issue was deeper down, and kind of bumped into it debugging the griddeddata tests.

**Comment by ejgal** [comment]
No worries! Glad it was a simple thing to fix.

**Comment by ejgal** [comment]
@jgliss it looks like auto closing issues only work when the commit is merged into the default branch.
Possible work around is to always set the current development branch as default.

**Comment by ejgal** [comment]
Changing the default branch to the current development branch will also make it simpler to identify merged branches that can be closed.

**Comment by jgliss** [comment]
@ejgal correct, but shouldn't they be closed only if they have been incorporated into master?

**Comment by ejgal** [comment]
True. I guess it depends on the type of issue.
